proto/interchain_security/ccv/provider/v1/tx.proto

```diff
@@ -30,18 +28,3 @@ message MsgAssignConsumerKey {
 }
 
 message MsgAssignConsumerKeyResponse {}
-
-// MsgRegisterConsumerRewardDenom allows an account to register
-// a consumer reward denom, i.e., add it to the list of denoms
-// accepted by the provider as rewards.
-message MsgRegisterConsumerRewardDenom {
```

Comment:
Note: Remember to tag the protos on https://buf.build/cosmos/interchain-security with the specific version this ends up in.